### PR TITLE
docs: scaffold additional documentation

### PR DESCRIPTION
**Pull request by nikku**
Follow up to https://github.com/bpmn-io/element-templates/pull/6 with more details on element template configuration + capabilities.

### CODE REVIEW

**Comment by nikku** [comment]
While this is not complete (yet) I'd still send this "upgrade package" out to be linked to.

File: README.md [hunk 1]
```diff
@@ -20,16 +22,97 @@ Through a [JSON document](https://docs.camunda.io/docs/next/components/modeler/d
...
+
+The basic setup requires a BPMN modeler as well as properties panel and element template provider modules:
+
+```javascript
+import BpmnModeler from 'bpmn-js/lib/Modeler';
+
+import {
+  BpmnPropertiesPanelModule,
+  BpmnPropertiesProviderModule,
+  ZeebePropertiesProviderModule,
```
Comment (barmac) [inline README.md:77]:
```suggestion
```

This is unnecessary as the templates provider already depends on Zeebe provider: https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/src/provider/cloud-element-templates/index.js#L18

We could also consider making Zeebe provider depend on the general BPMN one.

File: README.md [hunk 2]
```diff
@@ -20,16 +22,97 @@ Through a [JSON document](https://docs.camunda.io/docs/next/components/modeler/d
...
+
+```javascript
+const elementTemplates = [ ... ];
+
+const modeler = new BpmnModeler({
+  container: '#canvas',
+  additionalModules: [
+    BpmnPropertiesPanelModule,
+    BpmnPropertiesProviderModule,
+    ZeebePropertiesProviderModule,
```
Comment (barmac) [inline README.md:92]:
```suggestion
```

Same.

File: docs/API.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Create a new element from a template.
+
+### `ElementTemplates#applyTemplate(element, template): DiagramElement`
+
+Apply a template to an element (potentially replacing the element, applying property bindings, ...). Returns the element.
+
+### `ElementTemplates#getLatest(element|templateId): Template[]`
+
+Get element templates applicable to `element` or matching template with `templateId`
```
Comment (barmac) [inline docs/API.md:37]:
```suggestion
Get element templates applicable to `element` or matching template with `templateId`.
```

**Comment by barmac** [comment]
Looks good. Even if it's not finished yet, it is still an improvement of the documentation 🚀

File: README.md (hunk 1 above)
Comment (nikku) [reply README.md:77]: Let's leave this here still to show the bits required.

**Pull request merged**